app.rb

```diff
@@ -1,9 +1,33 @@
 require 'sinatra/base'
+require './lib/peeps.rb'
 
 class Chitter < Sinatra::Base
+  enable :sessions
+
   get '/test' do
     'Test page'
   end
 
+  get '/' do
```

Comment:
You did a great job of implementing the stretch user stories – nice work!  If you had some time to refactor, I'd suggest rolling up your three controller actions (relating to generating lists of peeps) into one, here... something like this...

```ruby
get '/' do
  if params[:search]
    @peeps = Peeps.search(params[:search])
  else
    @peeps = Peep.all
  end
  erb :index
end
```

(You'd also have to refactor your search form)